spec/models/reservation_spec.rb

```diff
@@ -529,6 +529,7 @@
         before do
           ScheduleRule.destroy_all
           @instrument.update_attributes(requires_approval: true)
+          @instrument.reload
```

Comment:
I think this might be better off as `@instrument.schedule_rules.reload` so it's a little more explicit about why we need to do it (and move it up a line). It's not actually the `update_attributes` that's causing the problem, it's that we're destroying ScheduleRules and  not telling `@instrument` about it and it already has the association loaded. Not sure why this was working before (maybe update_attributes used to do a `reload` implicitly?)